nibabel/streamlines/tests/test_streamlines.py

```diff
@@ -240,12 +244,17 @@ def test_save_complex_file(self):
                     nib.streamlines.save(complex_tractogram, filename)
 
                     # If streamlines format does not support saving data
-                    # per point or data per streamline, a warning message
+                    # per point or data per streamline, warning messages
                     # should be issued.
-                    if not (cls.SUPPORTS_DATA_PER_POINT and
-                            cls.SUPPORTS_DATA_PER_STREAMLINE):
-                        assert_equal(len(w), 1)
-                        assert_true(issubclass(w[0].category, Warning))
+                    nb_expected_warnings = 0
```

Comment:
I'll go with the following as I found the "for loop" a bit overkill for two elements.
```
nb_expected_warnings = \
    ((not cls.SUPPORTS_DATA_PER_POINT) +
    (not cls.SUPPORTS_DATA_PER_STREAMLINE))
```